rcl_action/src/rcl_action/goal_handle.c

```diff
@@ -68,6 +69,8 @@ rcl_action_goal_handle_init(
   goal_handle->impl->state = GOAL_STATE_ACCEPTED;
   // Copy the allocator
   goal_handle->impl->allocator = allocator;
+  // Set invalid time
+  goal_handle->impl->goal_terminal_timestamp = 0;
```

Comment:
0 is technically a valid time, I guess -1 or some define INVALID_TIME_POINT would be nicer.

Reply:
This value comes from the current clock time, so it can't be 0.  
Using macro definitions for unavailable values makes the code easier to understand.  
I want to use GOAL_TERMINAL_INVAILD_TIMESTAMP

Reply:
After checking rcl_get_ros_time(), rcl_get_steady_time() and rcl_get_system_time(), they should not return negative value.  So it is better to set INVAILD_GOAL_TERMINAL_TIMESTAMP as -1.